binder/src/main/java/io/grpc/binder/internal/BinderTransport.java

```diff
@@ -197,24 +198,14 @@
 
   @Nullable private IBinder outgoingBinder;
 
-  /** The number of outgoing bytes we've transmitted. */
-  private final AtomicLong numOutgoingBytes;
+  private final FlowController flowController = new FlowController(TRANSACTION_BYTES_WINDOW);
```

Comment:
nit: Allocate in constructor for consistency?

Reply:
done